### ISSUE TITLE
Current Country/Station API

### ISSUE THREAD

**Issue by N95JPL** [Koenvh1/ets2-local-radio#91]
Hello!
Would it be possible to add in an API call with the current Country/Radio Station with a URL such as
http://IPADDRESS:PORT/api/current-station/

For use with third-party applications? Mainly, I am building a Dashboard and have a radio display I would like to display the Radio I am on!

Thanks
N95JPL

**Comment by Koenvh1** [comment]
The server is not aware of what is being played, apart from showing the overlay. So whilst technically possible, it would be quite a hassle to implement.

**Comment by N95JPL** [comment]
Ah right okay, is there any way of saving it? Via a file, such as the Log?

**Comment by Koenvh1** [comment]
Yes there is. In main.js there are some global variables (g_current_country, g_current_station etc), plus you have access to the DOM. So what you could do is override the setRadioStation function in Javascript (using the custom.js file), get the data you need, pass it to somewhere (another server, some web service, your choice) and then call the original setRadioStation function. It's a bodge, but it will probably work.
